dev-util/shc/shc-3.9.6.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A generic shell script compiler"
+DESCRIPTION="Shc takes a script, which is specified on the command line and \
+produces C source code. The generated source code is then compiled and linked \
+to produce a stripped binary executable. \
+The compiled binary will still be dependent on the shell specified in the \
+first line of the shell code (i.e shebang) (i.e. #!/bin/sh), thus shc does not \
+create completely independent binaries."
+HOMEPAGE="https://github.com/neurobin/shc"
+COPYRIGHT="1997-2013 Francisco Garcia
+ 	2015-2017 Md Jahidul Hamid"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/neurobin/shc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="da6a2a3ff4c356a61e086c616561bf681489993cab00c426bad0cfd703a68063"
+SOURCE_FILENAME="shc-$portVersion.tar.gz"
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	shc = $portVersion
+	cmd:shc
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	"
```

Comment:
```diff
	cmd:make
 	"
+if [ "$recipeAction" = TEST ]; then
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:bash
+	"
+fi
```
Currently this is not really needed because bash is already our default shell, but it might be a good idea to have it here, in case we switch to a different shell.
That said, this is a lot of noise for something that is unlikely to happen. So please feel free to skip this suggestion :-)

Reply:
Wouldn't hurt to include it also :)